apollo初始化问题

(mypy3) ➜  automaticzabbix go run autozabbix.go 

[agollo] 2021/03/05 10:08:23 [INFO] start agollo client...
[agollo] 2021/03/05 10:08:23 [ERROR] fail to create cache dir: mkdir : no such file or directory
2021/03/05 10:08:23 apollo初始化失败： mkdir : no such file or directory
2021/03/05 10:08:23 

```
func (this *Obnfo) NewApollo() {
	conf := apollo.Conf{
		AppID:              this.Para.LocalInfo[0].Game.Code + "_config",
		Cluster:            "center",
		NameSpaceNames:     []string{"application"},
		AccesskeySecret:    "xxxxx",
		InsecureSkipVerify: false,
		MetaAddr:           "xxxx",
	}
	if err := apollo.Start(&conf); err != nil {
		log.Println("apollo初始化失败：", err)
	}
	value := apollo.GetString("root_pub")
	log.Println(value)
}
```

go版本：1.16

--- a/client.go
+++ b/client.go
@@ -357,6 +357,10 @@ func (c *client) setReleaseKey(namespace, releaseKey string) {
 
 // autoCreateCacheDir autoCreateCacheDir
 func (c *client) autoCreateCacheDir() error {
+	if c.conf.CacheDir == "" {
+		return nil
+	}
+
 	fs, err := os.Stat(c.conf.CacheDir)
 	if err != nil {
 		if os.IsNotExist(err) {

